<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="input_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"> .QLineEdit {
	background-color:#ffffff;
	border-radius:25px;
	border:2px solid #dcdcdc;
	color:#121212;
	font-family:Trebuchet MS;
	font-size:12px;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Enter your name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check_data">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>275</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
	background-color:#ffffff;
	border-radius:25px;
	border:2px solid #dcdcdc;
	color:#666666;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
}
.QPushButton:hover {
	background-color:#f6f6f6;
	color:#121212;
}
.QPushButton:pressed {
	border-color:#121212;
}</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_password">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"> .QLineEdit {
	background-color:#ffffff;
	border-radius:25px;
	border:2px solid #dcdcdc;
	color:#121212;
	font-family:Trebuchet MS;
	font-size:12px;
}
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Enter your password:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QRadioButton {
	color:#000000;
	font-family:Trebuchet MS;
	font-size:17px;
}

QRadioButton:hover{
	color:#121212;
 }
QRadioButton:checked{
	color:#666666;
 }
</string>
   </property>
   <property name="text">
    <string> Remember me?</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>-10</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#666666;
font-family:Bebas Neue;
font-size:40px;
</string>
   </property>
   <property name="text">
    <string>Qc</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="reg_info">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#121212;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Login: |FALSE|&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
